website/docs/d/atlas_user.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: atlas_user"
+sidebar_current: "docs-mongodbatlas-datasource-atlas-user"
+description: |-
+    Provides a Atlas User Datasource.
+---
+
+# Data Source: mongodbatlas_atlas_user
+
+`mongodbatlas_atlas_user` Provides a MongoDB Atlas User.
+
+-> **NOTE:** If you are the owner of a MongoDB Cloud organization or project, you can also retrieve the user profile for any user with membership in that organization or project.
+
+## Example Usage
+
+### Using user_id attribute to query
+```terraform
+data "mongodbatlas_atlas_user" "test" {
+  user_id = "<USER_ID>"
+}
+```
+
+### Using username attribute to query
+```terraform
+data "mongodbatlas_atlas_user" "test" {
+  username = "<USERNAME>"
+}
+```
+
+## Argument Reference
+
+* `user_id` - (Optional) Unique 24-hexadecimal digit string that identifies this user.
+* `username` - (Optional) Email address that belongs to the MongoDB Cloud user account. You cannot modify this address after creating the user.
+
+~> **IMPORTANT:** Either `user_id` or `username` must be configurated.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+* `country` - Two alphabet characters that identifies MongoDB Cloud user's geographic location. This parameter uses the ISO 3166-1a2 code format.
```

Comment:
```suggestion
* `country` - Two alphabet characters that identifies MongoDB Atlas user's geographic location. This parameter uses the ISO 3166-1a2 code format.
```

Reply:
question: can we link to a reference on that ISO format here?

Reply:
Would not be certain what to reference, this could be an option: https://www.iso.org/iso-3166-country-codes.html